llvm/utils/release/build_llvm_release.bat

```diff
@@ -7,14 +7,16 @@ goto begin
 echo Script for building the LLVM installer on Windows,
 echo used for the releases at https://github.com/llvm/llvm-project/releases
 echo.
-echo Usage: build_llvm_release.bat --version ^<version^> [--x86,--x64, --arm64]
+echo Usage: build_llvm_release.bat --version ^<version^> [--x86,--x64, --arm64] [--skip-checkout] [--local-python]

```

Comment:
the local-python flag doesn't seem to be used anywhere?

Reply:
I see it used on line 372.